Directory.Build.props

```diff
@@ -28,6 +26,12 @@
     <ArtifactsBinDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'bin'))</ArtifactsBinDir>
   </PropertyGroup>
 
+  <!-- The NetCoreApp TFM to build and test against. -->
+  <PropertyGroup>
+    <NetCoreAppCurrentTFMFull>.NETCoreApp,Version=v5.0</NetCoreAppCurrentTFMFull>
+    <NetCoreAppCurrentTFM>netcoreapp5.0</NetCoreAppCurrentTFM>
```

Comment:
Naming: can you consider `NetCoreAppCurrent` and `NetCoreAppCurrentTargetFrameworkMoniker`?

I feel that TFM is an unnecessary acronym to use here, and I prefer the brevity of `NetCoreAppCurrent` since I expect this property will appear **a lot** in the repository.

Reply:
Works for me.